### PR TITLE
Proposed content changes to the accessibility unit

### PR DESCRIPTION
**Pull request by smhigley**
❤️ having an accessibility section in a web dev course! I reviewed the content, and created the following suggested updates (with reasons detailed below):

- **Removed the word "physical" from "screen reader"**
- **Updated the wording about browser text reading tools**
    The reason for this is because the functionality and usage of tools like Edge's speak aloud feature and other text reading extensions are fundamentally different from screen readers. They are not primarily used for vision-related disabilities, and provide a very different UX and have different accessibility considerations than screen readers do. Since people who are new to accessibility sometimes confuse simpler text-reading tools for screen readers, I wanted to make the difference more clear in the tutorial wording.
- **Added a section on zoom**
  I thought zoom would fit really well in the tools section, since it's easy for anyone to try, and has wide accessibility benefits if done well. I also thought it would be a good place to include easy built-in browser accessibility tools, since all major browsers have some zoom and text sizing options built-in.
- **Removed the text highlight section**
  While some people will recommend using `<strong>` and `<em>` tags for emphasis because those tags have semantic meaning, it doesn't make a difference to screen reader users in practice, or to any other AT. It seems like a fine recommendation for developers learning HTML in general, bug I couldn't think of a way to relate it to accessibility, and I don't think it would be a good idea to imply that it makes a difference for screen reader accessibility.
- **Added heading hierarchy section**
  This was partly to make up for removing the section on text highlight, I wanted to add an alternative advice section on a type of semantic emphasis that makes a big difference to screen reader users.
- **Switched ARIA example from progressbar to tree**
  There is actually a semantic HTML element for progressbar, `<progress>`, so I updated the example to `role="tree"`
- **Some minor wording changes**, e.g. to make the distinction between a user and their assistive tech more clear

Thanks for your time, and please let me know if any of the changes aren't well explained or need to be updated!

### CODE REVIEW

**Comment by jlooper** [comment]
Thank you! Looks awesome to me, wdyt @GeekTrainer

File: 1-getting-started-lessons/3-accessibility/README.md [hunk 1]
```diff
@@ -127,24 +133,28 @@ You can use `aria-label` to describe the link when the format of the page doesn'
...
+<div role="tree" aria-labelledby="tree-label">
+  <div role="treeitem" aria-expanded="false" tabindex="0">Uploads</div>
 </div>
 ```
 
 ## Images
 
-It goes without saying screen readers are unable to automatically read what's in an image. Ensuring images are accessible doesn't take much work - it's what the `alt` attribute is all about. All images should have an `alt` to describe what they are.
+It goes without saying screen readers are unable to automatically read what's in an image. Ensuring images are accessible doesn't take much work - it's what the `alt` attribute is all about. All meaningful images should have an `alt` to describe what they are.
+
```
Comment (GeekTrainer) [inline 1-getting-started-lessons/3-accessibility/README.md:148]:
```suggestion
```

Can you collapse this into one paragraph?

**Comment by GeekTrainer** [comment]
I made the tiniest of suggestions about collapsing a couple of items into a single paragraph. Otherwise I think it looks fantastic!! And I learned something new about `strong`/`bold`. I'd always been taught it was for accessibility.

Thank you so much for the contributions!!

**Comment by jlooper** [comment]
thanks, everyone!

**Comment by smhigley** [comment]
Thanks so much for reviewing this and merging it so quickly! ❤

I may have one more a11y-related PR for you later, if that's ok 😄

**Comment by GeekTrainer** [comment]
Thank you so much for contributing!!!

**Pull request merged**